quarkus/runtime/src/main/java/org/keycloak/quarkus/runtime/cli/Picocli.java

```diff
@@ -434,8 +434,8 @@ && isUserModifiable(Configuration.getConfigValue(key))) {
                 throw new PropertyException(format("The following build time options have values that differ from what is persisted - the new values will NOT be used until another build is run: %s\n",
                         String.join(", ", ignoredBuildTime)));
             } else if (!ignoredRunTime.isEmpty()) {
-                warn(format("The following run time options were found, but will be ignored during build time: %s\n",
-                        String.join(", ", ignoredRunTime)), getOutWriter());
+                info(format("The following run time options were found, but will be ignored during build time: %s\n",
```

Comment:
I think it's fine to have it as an info only. Given it also helps with the test logic, let's keep as is.